tools/base/src/Base.ts

```diff
@@ -12,6 +12,7 @@ governing permissions and limitations under the License.
 
 import { LitElement, ReactiveElement } from 'lit';
 import { version } from '@spectrum-web-components/base/src/version.js';
+import { SpectrumDelegates } from '@spectrum-web-components/reactive-controllers/src/SpectrumDelegates.js';
```

Comment:
This causes a cyclical dependency. You need to make some decisions about where the types should live that allows a unidirectional dependency chain in order to allow this to build.